Security/centurylink-cloud-guide-to-vormetric-dsm.md

```diff
@@ -285,7 +285,7 @@ Policy Summary:
 ----------|------------|--------|-----------|----------|-----------
 **1**|&#42;|Trust_Set|&#42;|&#42;|Permit<br>Apply Key
 **2**|&#42;|&#42;|Backup_Set|Read/Write|Permit
-**3**|&#42;|&#42;|&#42;|&#42;|Deny<br>Audit
+**3**|&#42;|&#42;|&#42;|&#42;|Deny<br>Audit**
```

Comment:
Agreed. I did that so the Markdown would cooperate in Atom. Otherwise, Bold continued on for a couple paragraphs.